app/views/collections/_collection.html.erb

```diff
@@ -17,7 +17,8 @@
       <h5 class="card-title"><%= collection.name.titleize %></h5>
       <p class="card-text"><%= pluralize Model.tagged_with(collection, context: :collection).count, "model" %></p>
       <div>
-        <%= link_to "Open", collection_path(collection), {class: "btn btn-primary btn-sm"} %>
+        <%= icon "collection", "Collection" %> 
+        <%= link_to collection.name.titleize, params_passthrough().merge(controller: 'models', collection: collection.id) %>
```

Comment:
As this is in the collections list view, is params_passthrough relevant? Not sure it adds anything? Also, I liked the "open" button style a bit better than having a normal link... though I'm no expert at design, as you can tell :)

Reply:
Both of these are probably a cut/paste error here as I was extending out various pieces.